### DISCUSSION TITLE
Missing timestamps

### DISCUSSION THREAD

**Discussion by MorterzaAbbaszadeh** [NeurodataWithoutBorders/helpdesk#34] [Q&A]
Hey all, 

I was trying to access and navigate myself around DANDI 000016 ( here : https://dandiarchive.org/dandiset/000016?search=najafi&pos=1 )

there is a matlab live script demo for accessing different fields in the data:

https://github.com/datajoint-company/najafi-2018-matlab/blob/master/najafi_examples_with_matlab.mlx

i was able to open the nwb file and probe the subfields but I found that the data structure is different  compared to the one described on the access demo and I suspect the time stamps are missing. the demo accessed time stamps by:

`
event_roi_timeseries.get('Trial_00').timestamps`

but my attempt returned an empty array [].

is there any other way to store recover timestamps that perhaps I am missing or you guys 
I wrote a full diagnostic in the form of a live script, you can take a look in the PDF file below.

big thanks in advance

[diagniostic.pdf](https://github.com/NeurodataWithoutBorders/helpdesk/files/9629354/diagniostic.pdf)

**Comment by bendichter** [comment]
In NWB time series, you can store data timing either:
(a) by storing timestamps, which is an array of the same length of data 
or
(b) by specifying starting time (starting_time) and sampling rate (starting_time_rate).

You should be able to access these with `event_roi_timeseries.get('Trial_00').starting_time` and `event_roi_timeseries.get('Trial_00').starting_time_rate`
